pkgs/tools/admin/pgadmin/update.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl wget jq yq common-updater-scripts prefetch-yarn-deps yarn-lock-converter
+
+set -eu -o pipefail
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
```

Comment:
We don't need color in an update script

Reply:
1) It looks better
2)  there is a lot of output from the yarn-lock-converter, so it makes debugging any issues easier to spot

Does it hurt keeping the colors in?

Reply:
Not sure how well the logs of e.g. r-ryantm look. Maybe they contain the plain escape characters.

Can we mute some of that output? Maybe redirect stdout to /dev/null?